login/service.go

```
@@ -0,0 +1,76 @@
+package login
+
+import (
+	"bankapp/config"
+	"bankapp/db"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	"go.uber.org/zap"
+)
+
+var secretKey = []byte(config.JWTSignature())
+
+type Service interface {
+	login(ctx context.Context, req loginRequest) (tokenString string, err error)
+}
+
+type loginService struct {
+	store  db.LoginStorer
+	logger *zap.SugaredLogger
+}
+
+type Claims struct {
+	Email string `json:"email"`
+	Role  int    `json:"role"`
+	jwt.StandardClaims
+}
+
+func (ls *loginService) login(ctx context.Context, ul loginRequest) (tokenString string, err error) {
+	fmt.Println("ul is --> ", ul.Email)
+	user, err := ls.store.FindUserByEmail(ctx, ul.Email)
+	// TODO: Handle the err
```

don't ignore errors. While writing code at first time